client/src/components/reader/Header.js

```diff
@@ -23,9 +24,21 @@ export default class Header extends Component {
     incrementMargins: PropTypes.func,
     decrementMargins: PropTypes.func,
     setColorScheme: PropTypes.func,
+    scrollAware: PropTypes.object,
     startLogout: PropTypes.func
   };
 
+  getSectionTitle(id) {
```

Comment:
To me, the way this is being done has a bad smell. Traversing the TOC to get the title of a section for display in the header is wrong, since the section model has its title stored on it. In looking at the code, I can see why you're doing this—the reader container hasn't fetched the section from the section from the store, so you don't have access to the model. Rather than working around it as you've done here, the better solution would be to do some light refactoring. Instead of finding an indirect way to get the section in the header, I suspect we should move the section fetching to the reader container.

I'll make a story to do this refactoring, and put it in a separate commit.